README.md

```diff
@@ -25,37 +25,85 @@ pip install spotipy --upgrade
 A full set of examples can be found in the [online documentation](http://spotipy.readthedocs.org/) and in the [Spotipy examples directory](https://github.com/plamere/spotipy/tree/master/examples).
 
 To get started, install spotipy and create an app on https://developers.spotify.com/.
+
+To create an app on https://developers.spotify.com/ and obtain the Client ID and Client Secret.
+1. On the homepage, click on "Dashboard".
+2. Login or create a Spotify account.
+3. Click on "Create An App".
+4. Click on your created app.
+5. Copy the "Client ID" and "Client Secret".
+
 Add your new ID and SECRET to your environment:
 
 ### Without user authentication
 
 ```python
+"""
+Importing allows the user to use Spotipy API to obtain a token from a Spotify profile to be used within the code
+By using Spotipy, a user is able to obtain the necessary authentication information from a Spotify account and create
+an object to be stored into a variable. 
+"""
 import spotipy
 from spotipy.oauth2 import SpotifyClientCredentials
 
+"""
+Without user authentication, the program immediately displays the information the program asks Spotify to provide. 
+client_id and client_secret are the only information needed to access a Spotify account. Below Spotipy will obtain a token 
+to authenticate and create an object the user can use. 
+"""
```

Comment:
I appreciate the effort on commenting all the code, however this is too detailed and makes it difficult to see the code quickly and compare the two different modes: with and without user authentication.

I think most developers can guess what each line is doing, and prefer doing so rather than read too much english.

If you remove these code comments I would be happy to merge the PR